### PR TITLE
AU updates

### LINKED ISSUE #100: CCDB AU/003 - Section 1.3.2
### Comment:
The acronym entry for CA row is Certificate Authority but "Client Application" is used in the cPP and is the meaning when "S.CA" is used in this document.

### Proposal:
Change acronym table entry for CA to "Client Application".

### Priority: 1

### LINKED ISSUE #101: CCDB AU/004 - Section 2
Para 17

### Comment:
Section 2 is titled Evaluation Activities for SFRs but doesn’t mention or clearly reference the KMD.  The KMD is explained much later in Section 6.

### Proposal:
Perhaps modify Para 17 to introduce KMD and explicitly reference Section 6 where it is explained in more detail.

### Priority: 1

### PR DESCRIPTION
**Pull request by woodbe**
This is to close #100 and close #101.

### CODE REVIEW

File: SD_DSC.adoc [hunk 1]
```diff
@@ -249,7 +249,7 @@ For definitions of standard CC terminology, see [CC] part 1.
 |Advanced Encryption Standard
 
 |CA 
-|Certificate Authority
+|CClient Application
```
Comment (yiatsec) [inline SD_DSC.adoc:252]: Should "CClient" be "Client"?

**Review by yiatsec — COMMENTED** [review]
I believe "CClient Application" should be "Client Application."

File: SD_DSC.adoc (hunk 1 above)
Comment (smuellerDD) [inline SD_DSC.adoc]: CClient -> typo?

**Comment by smuellerDD** [comment]
Apart from a small typo, the change looks good to me.

File: SD_DSC.adoc (hunk 1 above)
Comment (woodbe) [reply SD_DSC.adoc]: this was fixed in the second commit

**Pull request merged**